src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,18 @@ title = "Releases"
 weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
+## v0.14.1
+
+This is a bugfix release:
+
+* Stop using cluster-owned tag for AWS Security Group lookup.
+* Running the `subctl diagnose firewall` command with individual kubeconfigs will now override the default namespace to `submariner-operator`.
```

Comment:
```suggestion
* Running the `subctl diagnose firewall` command with individual kubeconfigs will now deploy diagnose pods in the  `submariner-operator` namespace to avoid podSecurity errors.
```